.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Your project configuration should be kept separate from the source code. Please put it in .gitignore

Reply:
done.
.project is added to .gitignore file.

Thanks,
Reza